py-polars/polars/dependencies.py

```diff
@@ -219,6 +223,7 @@ def _check_for_pydantic(obj: Any) -> bool:
     "pickle",
     "subprocess",
     # lazy-load third party libs
+    "dataframe_api_compat",
```

Comment:
I think this should then also become part of our development dependencies / extras defined in pyproject.toml, and maybe include a very simple test for each just to show that the methods work / don't get deleted accidentally?

Reply:
good idea, thanks - I've added to `requirements-dev.txt`. Leaving it out of `pyproject.toml` for now, perhaps it can be added there once it's more mature?

Reply:
Fair enough!